frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/VerifyCodeHandler.java

```diff
@@ -263,7 +270,17 @@ private boolean checkErrorCountsForReauth(
                     ReauthAuthenticationAttemptsHelper.countTypesWhereUserIsBlockedForReauth(
                             countsByJourney, configurationService);
 
+            ClientRegistry client = userContext.getClient().orElse(null);
             if (!countTypesWhereBlocked.isEmpty()) {
+                String rpPairwiseId =
+                        getInternalCommonSubjectIdentifier(userProfileMaybe.get(), client);
```

Comment:
If this is called with client having a value of null then you will get a NullPointerException.  If we know client is null then there is no point making this call as we are in an error state.